From 09cd7025f9977b79fe3df9ae7a80c4c3f7dc6327 Mon Sep 17 00:00:00 2001
From: tanghui <tanghui20@huawei.com>
Date: Mon, 11 Apr 2022 15:33:46 +0800
Subject: [PATCH 0794/2065] [Huawei] config: enable
 CONFIG_QOS_SCHED_DYNAMIC_AFFINITY by default

Offering: HULK
hulk inclusion
category: feature
bugzilla: 186575

--------------------------------

Signed-off-by: tanghui <tanghui20@huawei.com>
---
 arch/arm64/configs/openeuler_defconfig | 1 +
 arch/x86/configs/openeuler_defconfig   | 1 +
 2 files changed, 2 insertions(+)

diff --git a/arch/arm64/configs/openeuler_defconfig b/arch/arm64/configs/openeuler_defconfig
index 770222a597e4..a4fd4cdcbd3e 100644
--- a/arch/arm64/configs/openeuler_defconfig
+++ b/arch/arm64/configs/openeuler_defconfig
@@ -151,6 +151,7 @@ CONFIG_CGROUP_CPUACCT=y
 CONFIG_CGROUP_PERF=y
 CONFIG_CGROUP_BPF=y
 CONFIG_QOS_SCHED=y
+CONFIG_QOS_SCHED_DYNAMIC_AFFINITY=y
 # CONFIG_CGROUP_DEBUG is not set
 CONFIG_SOCK_CGROUP_DATA=y
 CONFIG_CGROUP_FILES=y
diff --git a/arch/x86/configs/openeuler_defconfig b/arch/x86/configs/openeuler_defconfig
index 926dfe0628dc..918de84d99f3 100644
--- a/arch/x86/configs/openeuler_defconfig
+++ b/arch/x86/configs/openeuler_defconfig
@@ -157,6 +157,7 @@ CONFIG_BLK_CGROUP=y
 CONFIG_CGROUP_WRITEBACK=y
 CONFIG_CGROUP_SCHED=y
 CONFIG_QOS_SCHED=y
+CONFIG_QOS_SCHED_DYNAMIC_AFFINITY=y
 CONFIG_FAIR_GROUP_SCHED=y
 CONFIG_CFS_BANDWIDTH=y
 CONFIG_RT_GROUP_SCHED=y
-- 
2.17.1

